### ISSUE TITLE
BluetoothMedic crashes if Bluetooth off

### ISSUE THREAD

**Issue by davidgyoung** [AltBeacon/android-beacon-library#668]
### Expected behavior

Does not crash

### Actual behavior

If Bluetooth is off, you get this:

```
 FATAL EXCEPTION: BluetoothTestThread
04-10 01:04:42.179  5703  8356 E AndroidRuntime: Process: com.xxxxxxx.app, PID: 5703
04-10 01:04:42.179  5703  8356 E AndroidRuntime: java.lang.IllegalStateException: BT Adapter is not turned ON
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.bluetooth.le.BluetoothLeUtils.checkAdapterStateOn(BluetoothLeUtils.java:136)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.bluetooth.le.BluetoothLeScanner.startScan(BluetoothLeScanner.java:204)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.bluetooth.le.BluetoothLeScanner.startScan(BluetoothLeScanner.java:132)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.bluetooth.le.BluetoothLeScanner.startScan(BluetoothLeScanner.java:113)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at org.altbeacon.bluetooth.BluetoothMedic.runScanTest(BluetoothMedic.java:261)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at org.altbeacon.bluetooth.BluetoothTestJob$1.run(BluetoothTestJob.java:104)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.os.Handler.handleCallback(Handler.java:790)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.os.Handler.dispatchMessage(Handler.java:99)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.os.Looper.loop(Looper.java:164)
04-10 01:04:42.179  5703  8356 E AndroidRuntime:    at android.os.HandlerThread.run(HandlerThread.java:65)
```

### Steps to reproduce this behavior

Unknown

### Mobile device model and OS version

Pixel 2

### Android Beacon Library version

Latest

IMPORTANT:  This forum is reserved for feature requests or reproducible bugs with the library itself.  If you need help with using the library with your project, please open a new question on StackOverflow.com.

**Comment by davidgyoung** [comment]
An easy solution is to simply catch the IllsegalStateException when starting scanning

**Comment by davidgyoung** [comment]
Potentially related: #672

**Comment by davidgyoung** [comment]
Fixed in #675
